### PR TITLE
CIV:6054:Claimant Intent - Part admit + pay set by date - How they want to pay.

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-6054

### Change description ###
added a new screen for claimantIntent : Part admit + set by date scenario.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/routes/features/claimantResponse/reviewDefendantsResponseController.ts [hunk 1]
```diff
@@ -33,4 +33,26 @@ reviewDefendantsResponseController.get(CLAIMANT_RESPONSE_REVIEW_DEFENDANTS_RESPO
...
+    const lang = req.query.lang ? req.query.lang : req.cookies.lang;
+    const claim: Claim = await getCaseDataFromStore(claimId);
+    const continueLink = constructResponseUrlWithIdParams(claimId, CLAIMANT_RESPONSE_TASK_LIST_URL);
+    if (claim?.responseStatus === ClaimResponseStatus.PA_NOT_PAID_PAY_BY_DATE) {
+      const financialDetails = getFinancialDetails(claim, lang);
+      res.render('features/claimantResponse/how-they-want-to-pay-response', {
+        claim,
+        continueLink,
+        financialDetails,
+        paid_by_date: formatDateToFullDate(claim.partialAdmission.paymentIntention.paymentDate),
```
Comment (laperla11) [inline src/main/routes/features/claimantResponse/reviewDefendantsResponseController.ts:48]: pls use camel case here

File: src/main/views/features/claimantResponse/how-they-want-to-pay-response.njk [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+{% block content %}
+  <div class="govuk-width-container">
+    <main class="govuk-main-wrapper" role="main">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="govuk-heading-l">{{ t('PAGES.FULL_ADMIT_SET_DATE_PAYMENT.TITLE') }}</h1>
+          <p class="govuk-body">
+            {{ t('PAGES.REVIEW_DEFENDANTS_RESPONSE.HOW_THEY_WANT_TO_PAY_RESPONSE',{
+              amount: claim.partialAdmission.howMuchDoYouOwe.amount,
+              by_date: paid_by_date}) }}
```
Comment (laperla11) [inline src/main/views/features/claimantResponse/how-they-want-to-pay-response.njk:21]: pls use camel case here

**Review by laperla11 — CHANGES_REQUESTED** [review]

File: src/main/routes/features/claimantResponse/reviewDefendantsResponseController.ts (hunk 1 above)
Comment (neeta-hmcts) [reply src/main/routes/features/claimantResponse/reviewDefendantsResponseController.ts]: done

File: src/main/views/features/claimantResponse/how-they-want-to-pay-response.njk (hunk 2 above)
Comment (neeta-hmcts) [reply src/main/views/features/claimantResponse/how-they-want-to-pay-response.njk]: done

**Review by laperla11 — APPROVED** [review]

**Pull request merged**